src/NzbDrone.Core/Indexers/RssSyncService.cs

```diff
@@ -44,7 +51,7 @@ private ProcessedDecisions Sync()
         {
             _logger.ProgressInfo("Starting RSS Sync");
 
-            var reports = _rssFetcherAndParser.Fetch().Concat(_pendingReleaseService.GetPending()).ToList();
+            var reports = _rssFetcherAndParser.Fetch().Concat(FilterByIndexer(_pendingReleaseService.GetPending())).ToList();
```

Comment:
I would suggest `var pendingReleases` and `var liveReports` and concating them into `reports`.

also it's a bit strange that  `RssFetcherAndParser` is dealing with indexer backoff, but `PendingReleaseService` doesn't. it means `RssSyncService` knows a bit too much about their internals.